src/test/java/FooterTest.java

```diff
@@ -0,0 +1,94 @@
+import components.Footer;
+import enums.Attribute;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import utils.WebDriverUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class FooterTest {
+    private static final String HOMEPAGE_URL = "https://magento.softwaretestingboard.com/";
```

Comment:
"https://magento.softwaretestingboard.com/" is used in all class. 
It would be great to store it in one place as public static final field ( in a new class like `PageUrl`) and reuse it in all test classes.